xikolo-ios/Views/NewsArticleCell.swift

```diff
@@ -13,11 +13,28 @@ class NewsArticleCell : UITableViewCell {
     @IBOutlet weak var titleView: UILabel!
     @IBOutlet weak var descriptionView: UILabel!
     @IBOutlet weak var readStateView: UIView!
+    @IBOutlet weak var dateView: UILabel!
+    @IBOutlet weak var courseView: UILabel!
 
     func configure(newsArticle: NewsArticle) {
         readStateView.backgroundColor = Brand.TintColor
+        
+        let dateFormatter = NSDateFormatter()
```

Comment:
Why? It tells me that `NSDateFormatter()` is not an Optional, so I shouldn't put it inside an `if let`.